src/windows_stubs.cr

```diff
@@ -89,11 +89,3 @@ end
 enum Signal
   KILL = 0
 end
-
-def sleep(seconds : Number)
```

Comment:
The removal of these two sleep overloads is unrelated to this PR, isn't it? Still, if it's some clean up, I don't oppose to it, but it took me by surprise.

Reply:
No, this is actually the purpose of this PR: We don't need the windows stubs anymore.

(Well, in reality, they haven't been in use anymore, because they are overridden by the ones in `concurrent.cr` since #7995. So technically, these removals could be considered a cleanup...)